pkgs/servers/asterisk/default.nix

```diff
@@ -1,59 +1,51 @@
-{ stdenv, pkgs, fetchurl, fetchgit,
+let versions = {
+  stable = ./stable.nix;
+  lts = ./lts.nix;
+};
+in
+{ stdenv, pkgs,
```

Comment:
I think this is unusual style. The only differences between the two versions (lts & stable) are version and hash. So it would be smarter to switch inside one nix-file, compare for example PostgreSQL https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/sql/postgresql/default.nix